### PR TITLE
Improved and updated readme

### PR DESCRIPTION
**Pull request by AlbertoElias**
Addresses #22

### CODE REVIEW

File: readme.html [hunk 1]
```diff
@@ -0,0 +1,1089 @@
+<!DOCTYPE html><html><head><meta charset="utf-8"><style>body {
```
Comment (kaelig) [inline readme.html]: Looks like the built file was committed. Is that normal?

File: readme.md [hunk 2]
```diff
@@ -106,3 +65,45 @@ Compiles to:
...
+```
+
+## Checking font rendering
+
+Different browsers use different font formats. Different browsers use different font formats. We only support `.woff` which is supported on all major browsers (>=IE9). You should make sure the demos for the new font family load properly on:
+
+* Chrome
+* IE >9
+* Firefox
+* Safari
```
Comment (kaelig) [inline readme.md]: This is quite ambiguous, maybe precise a version number?

**Comment by AlbertoElias** [comment]
@kaelig better?

**Comment by kaelig** [comment]
For the fonts table, what do you think of something simpler, like this?

| BentonSans | MillerDisplay |     MetricWeb    |
|:----------:|:-------------:|:----------------:|
|            |               |       thin       |
|    light   |               |       light      |
|   regular  |    regular    |      regular     |
|            |               | regular (italic) |
|            |               |      medium      |
|            |               |     semibold     |
|    bold    |      bold     |       bold       |
|            |               |   bold (italic)  |
|            |     black     |                  |

**Comment by AlbertoElias** [comment]
I'm using your wording but I think a simple list per column is much easier to maintain, markdown tables are quite a bit of a pain

**Comment by kaelig** [comment]
Sorry to insist, but documenting font stacks properly is something I've found important to spend time on at the Guardian. Tables are hard to maintain regardless of the markup language, unfortunately.

We can now put them all on the same table:

| BentonSans | MillerDisplay | MetricWeb | FinancierDisplayWeb | FinancierTextWeb |
|:-:|:-:|:-:|:-:|:-:|
| | | thin |
| light | | light |
| regular | regular | regular | regular | regular |
| | | regular (italic) | regular (italic) | regular (italic) |
| | | medium |
| | | semibold |
| bold | bold | bold |
| | | bold (italic) |
| | black |

The markup isn't very readable but I think the output makes more sense that way.

Do you think it helps conveying the information better?

**Comment by kaelig** [comment]
:+1:

**Pull request merged**